packages/ember-intl/package.json

```diff
@@ -97,9 +97,13 @@
     "webpack": "^5.94.0"
   },
   "peerDependencies": {
+    "@ember/test-helpers": "^3.2.0 || ^4.0.0",
     "typescript": "^5.0.0"
   },
   "peerDependenciesMeta": {
+    "@ember/test-helpers": {
+      "optional": false
```

Comment:
This should likely be `optional: true`, because you can't guarantee that your consumers are all going to use the testing util which depends on this package.

Reply:
Thanks for the suggestion. I think it makes sense to set `optional` to `true` for `@ember/test-helpers`. While we can assume that `@ember/test-helpers` is always present in apps and v1 addons, it may not be in v2 addons that only use helpers from `ember-intl`.